### PR TITLE
Update doc and add MLflow integration section

### PR DESCRIPTION
**Pull request by CGuichard**
Update screenshots, re-structure the navigation, add API reference.

Add section on ML Tracking with GitLab and MLflow SharingHub.

### CODE REVIEW

File: src/design/architecture.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+- **SharingHub UI**:
+
+The "SharingHub UI" is the main entry point. It takes the form of a web portal specially developed to make it easy to discover, browse,
+review and download services, code, AI models and datasets.
+It dynamically extracts the information stored by users and present it in the best possible way once it has been developed/improved.
+
+- **SharingHub Server**:
+
+The "SharingHub Server" is the engine that support all the intelligence that enables the portal to carry out its operations.
```
Comment (okoko-cs) [inline src/design/architecture.md]: supports

File: src/explore/project-view.md [hunk 2]
```diff
@@ -1,36 +1,73 @@
...
+### STAC
+
+The STAC helper display download code with the STAC API by using Python `requests` and EODAG, as well as STAC informations.
+
+![STAC helper](../assets/figures/explore/project-view/stac-helper.png)
+
+### DVC
+
+The DVC button displays configuration and setup of DVC for the project.
+The DVC service enables you to store large volumes of data. The DVC button is therefore only displayed for projects in the "Datasets" category, which have a DVC configuration in the source GitLab project.  This button displays the remote DVC configuration link and additional information on the DVC [documentation](https://dvc.org/doc) and [tutorial](../tutorials/manage_dataset_with_dvc.md).
```
Comment (okoko-cs) [inline src/explore/project-view.md]: AI model with big *pickle or joblib can not use DVC ?

File: src/share/ml-tracking/mlflow-sharinghub.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- Update: Owner, Maintainer, Developer
+- Delete: Owner, Maintainer, Developer
+
+## Authentication
+
+MLflow SharingHub uses SharingHub authentication session, meaning that **if you login in SharingHub, you should also be authenticated in MLflow SharingHub**.
+
+## Limitations
+
+It is important to note that there are some _limitations_ and _constraints__ on MLflow SharingHub. It is a plugin, on _top_ of MLflow, meaning MLflow particularities impose restrictions. Despite having multiple tracking URIs, we have a single server and database. In MLflow, experiments names, and model names (model registry) are _unique_, if I name an experiment, no one can create another experiment with the same name.
```
Comment (okoko-cs) [inline src/share/ml-tracking/mlflow-sharinghub.md]: _constraints__ -> _constraints_

File: src/explore/search.md [hunk 4]
```diff
@@ -19,13 +19,13 @@ There are two types of tags: **common tags**, which refers to the most commonly
 
 The search by title field allows you to dynamically filter projects in the current category page.
 
-![filter by title](../assets/figures/explore/search/filter_by_title.png)
+![filter by title](../assets/figures/explore/search/filter-by-title.png)
 
 ## Filter by starred projects
 
 A a category page, the star button allows to filter only projects you have starred. Projects of any categories can be starred independently on SharingHub or GitLab and this status will be automatically synchronized between both.
```
Comment (okoko-cs) [inline src/explore/search.md]: Is not clear here, is it possible begin sentence with "the star button allows to filter only projects you have starred."

File: src/explore/project-view.md (hunk 2 above)
Comment (CGuichard) [reply src/explore/project-view.md]: Excellent question, that needs further discussions. I am currently investigating this topic, named "Where do we put our models ?".

But for the moment I think that saying DVC is for datasets should not pose any problem.

File: src/explore/search.md (hunk 4 above)
Comment (okoko-cs) [reply src/explore/search.md]: @CGuichard this review is pending

**Review by okoko-cs — APPROVED** [review]

**Pull request merged**